qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -417,6 +417,12 @@
                 "province_state",
                 "county"
               ]
+            },
+            {
+              "title": "Location",
```

Comment:
lgtm but do we want a new tab? Gina can confirm that

Reply:
you are right, @gkuffel has confirmed we should move that field to an existing tab